### ISSUE TITLE
Basic authentication on webpage

### ISSUE THREAD

**Issue by jeroenst** [tzapu/WiFiManager#1681]
Is it possible to set basic authentication for the webpage?

I want to keep the portal running always but it has to be protected against unwanted visitors.

**Comment by hurricanefrog** [comment]
Yes, it does. You have to manually adjust the source code, though.

(WiFiManager.cpp, function void WiFiManager::handleRequest(), line 1301 ff.)
